examples/platforms/utils/link_metrics.cpp

```diff
@@ -257,4 +261,11 @@ uint8_t otLinkMetricsEnhAckGetDataLen(const otMacAddress *aMacAddress)
 exit:
     return len;
 }
+
+void otLinkMetricsResetEnhAckProbing()
+{
+    // clean the pool and delete all the content of the list
```

Comment:
Suggest removing the comment  as it repeats what the code does and code is clear :)
```suggestion
```